/*
The order in the list will determine the order of the project

"researchTitle" is the name of the project
"researchLead" is the leader of the project (likely one of the prinicipal investigators
"researchImage" is the link to the researchImage
"researchDescription" is the description of the project
"researchLinks" is the link to the publications
    Note, the publication link will be used to match with the publication information from the publication.js
*/
export const research = 
[
    {
        "researchTitle": "Plasma proteomics",
        "researchLead": "Maria Pernemalm",
        "researchImage": "mariaresearchpicture.jpg",
        "researchDescription": "<ul style='padding-left:20px'><li style='list-style-type:disc'>Goal: Develop an accurate and sensitive pipeline for mass spectrometry-based analysis of the plasma proteome.</li><li style='list-style-type:disc'>Challenge: Difficulty in detecting protein biomarkers in plasma due to skewed protein concentration distribution and high patient-to-patient variability.</li><li style='list-style-type:disc'>Methodology: Developing HiRIEF-based methodologies for global proteome analysis and targeted MRM-analysis of the plasma proteome.</li><li style='list-style-type:disc'>Incorporating Advances: Utilizing proteogenomics field advancements, including data on single amino acid variants in analyses.</li><li style='list-style-type:disc'>Applications:<ul style='padding-left:20px'><li style='list-style-type:disc'>Discovering diagnostic, prognostic, and predictive biomarkers in plasma.<li>Analyzing the protein corona of nanoparticles.</li><li style='list-style-type:disc'>Quantifying proteins from immunocapture analysis.</li><li style='list-style-type:disc'>Analyzing biological fluids such as pleural effusion, cystic fluid, and cerebrospinal fluid.</li></ul></ul>",
        "researchLinks": ["https://doi.org/10.1002/1878-0261.13355","https://doi.org/10.1186/s12931-023-02364-y","https://doi.org/10.3389/fendo.2022.971313"]
    },
    {
        "researchTitle": "Lung cancer",
        "researchLead": "Lukas Orre",
        "researchImage": "lukasresearchpicture.jpeg",
        "researchDescription": "<ul style='padding-left:20px'><li style='list-style-type:disc'>Objective: Identify novel biomarkers and targets for cancer therapy, specifically for non-small cell lung cancer (NSCLC).</li><li style='list-style-type:disc'>Focus: Study the effects of cancer therapy, particularly therapies targeting receptor tyrosine kinases like epidermal growth factor receptor (EGFR).</li><li style='list-style-type:disc'>Approach:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Develop and apply advanced methods to understand the molecular response of cancer cells to different drugs.</li><li style='list-style-type:disc'>Investigate determinants of drug response or resistance to suggest biomarkers and optimal drug combinations for improved lung cancer therapy.</li></ul><li style='list-style-type:disc'>Importance of Protein Expression and Function:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Proteins in a cell determine its phenotype, including whether it is cancerous and its sensitivity to specific treatments.</li><li style='list-style-type:disc'>Utilize proteomics for large-scale study of proteins in biological systems.</li></ul><li style='list-style-type:disc'>Functional Proteomics:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Develop and apply proteomics methods to describe the functional state of proteins, including information about protein activity (e.g., phosphorylation), protein complex formation, and protein subcellular location.</li></ul><li style='list-style-type:disc'>Purpose:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Understand why cells from different tumors respond differently to cancer drugs.</li><li style='list-style-type:disc'>Identify personalized combinations of cancer therapies for each patient.</li></ul><li style='list-style-type:disc'>Long-term Goal: Tailor the best therapy for each lung cancer patient based on molecular understanding and personalized treatment approaches.</ul>",
        "researchLinks": ["https://doi.org/10.1016/j.molcel.2018.11.035","https://doi.org/10.1038/s41596-022-00699-2","https://doi.org/10.1002/jev2.12128"]
    },
    {
        "researchTitle": "Breast cancer",
        "researchLead": "Henrik Johansson",
        "researchImage": "henrikresearchpicture.jpg",
        "researchDescription": "<ul style='padding-left:20px'><li style='list-style-type:disc'>Team's Overall Aim:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Use MS-based proteomics tools to enhance biological understanding of breast cancer.</li><li style='list-style-type:disc'>Define clinical subtypes and identify biomarkers for personalized therapy.</li></ul><li style='list-style-type:disc'>Clinical Stratification:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Estrogen and progesterone receptors, along with HER2, are routinely used for breast cancer patient stratification.</li><li style='list-style-type:disc'>Genomic efforts (PAM50 subtypes) further classify breast cancer into Luminal A, B, HER2, Basal, and normal-like subtypes based on mRNA expression and mutational patterns.</li></ul><li style='list-style-type:disc'>Tamoxifen Treatment:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Tamoxifen is a key treatment for estrogen receptor-positive breast cancer for nearly four decades.</li><li style='list-style-type:disc'>Reduces recurrence rate by 50%, but about one-third of patients still relapse.</li><li style='list-style-type:disc'>80% of breast cancer patients are eligible for tamoxifen, emphasizing the need for biomarkers to guide treatment.</li><li style='list-style-type:disc'>Identified a 13-protein panel and the nuclear receptor RARA as potential predictive markers of tamoxifen resistance.</li></ul><li style='list-style-type:disc'>Proteomics Method:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Use MS-based proteomics methods involving isoelectric focusing of peptides followed by MS analysis for in-depth proteome quantification.</li><li style='list-style-type:disc'>Combine proteome information with other data levels (copy number, mRNA) to gain a comprehensive understanding of breast cancer biology.</li><li style='list-style-type:disc'>Explore the link between mRNA and protein, recognizing the limitations of a direct correlation.</li></ul><li style='list-style-type:disc'>Biomarker Exploration:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Investigate protein interactions as functional unit biomarkers in addition to protein levels.</li><li style='list-style-type:disc'>Utilize proteomics to identify novel biomarkers for personalized therapy.</li></ul><li style='list-style-type:disc'>Mutational Landscapes and Immunotherapy:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Explore how mutational landscapes affect the proteome.</li><li style='list-style-type:disc'>Investigate the potential use of mutational information as tumor-specific antigens in immunotherapy.</li></ul></ul>",
        "researchLinks": ["https://doi.org/10.1158/2767-9764.CRC-21-0100","https://doi.org/10.1038/s43018-021-00259-9","https://doi.org/10.21203/rs.3.pex-1565/v1"]
    },
    {
        "researchTitle": "Proteomics methods",
        "researchLead": "Rui Branca",
        "researchImage": "ruiresearchpicture.png",
        "researchDescription": "<ul style='padding-left:20px'><li style='list-style-type:disc'>Group's Focus:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Develop and apply MS-based methods to enhance human proteome analysis.</ul></li><li style='list-style-type:disc'>Prefractionation Method:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Developed high-resolution peptide isoelectric focusing (HiRIEF) method for reproducible fractionation before MS-analysis.</li><li style='list-style-type:disc'>Improves analytical depth, increases peptide and protein sequence coverage, and enhances overlap between samples.</li><li style='list-style-type:disc'>Provides additional data point: peptide isoelectric point, enabling novel applications.</li></ul><li style='list-style-type:disc'>Proteogenomics:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Applied MS-data in an unbiased genome-wide search for protein coding DNA sequences in the human genome.</li><li style='list-style-type:disc'>Aims to integrate genomics and proteomics data for a comprehensive understanding.</li></ul><li style='list-style-type:disc'>Tissue and Plasma Proteome Analysis:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Developing methods to improve analysis in tissues and plasma.</li><li style='list-style-type:disc'>Focus on detecting and quantifying post-translational modifications, including phosphorylations, glycosylations, and redox-related cysteine modifications.</li></ul><li style='list-style-type:disc'>Targeted Analysis:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Developing methods for targeted analysis of proteins to enhance specificity.</li></ul><li style='list-style-type:disc'>Bioinformatics Tools:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Actively working on the development and use of bioinformatics tools to interpret vast amounts of proteomics data.</li></ul><li style='list-style-type:disc'>Recent Tool Developments:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Novel tools for splice variant-specific quantification at the protein level.</li><li style='list-style-type:disc'>Galaxy-based proteogenomics pipeline.</li><li style='list-style-type:disc'>Methods to improve quantification accuracy in proteomics.</li></ul><li style='list-style-type:disc'>Overall Goal:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Continuous improvement and innovation in MS-based methods and bioinformatics tools for comprehensive and accurate human proteome analysis.</li></ul></ul>",
        "researchLinks": ["https://doi.org/10.1038/s41467-019-09018-y","https://doi.org/10.1038/s41467-018-03311-y","https://doi.org/10.7554/eLife.71156"]
    },
    {
        "researchTitle": "Childhood cancer",
        "researchLead": "Rozbeh Jafari",
        "researchImage": "rozbehresearchpicture.png",
        "researchDescription": "<ul style='padding-left:20px'><li style='list-style-type:disc'>Team's Overall Aim:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Use mass spectrometry-based proteomics to enhance biological understanding of Acute Lymphoblastic Leukemia (ALL) and Neuroblastoma.</li>Define clinical subtypes, evaluate drug interactions, and identify biomarkers for targeted therapies, with a focus on improving personalized therapy.</li></ul><li style='list-style-type:disc'>Childhood ALL Significance:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Childhood ALL is the most common pediatric cancer, representing 75-80% of all childhood cancer cases.</li><li style='list-style-type:disc'>While many patients achieve complete remission, relapsed cases have a poor prognosis.</li><li style='list-style-type:disc'>Current broad cytotoxic therapy for relapsed ALL has significant resistance development and long-term side effects.</li></ul><li style='list-style-type:disc'>Treatment Challenges:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Relapsed ALL treatment is more intense, significantly impacting the quality of life.</li><li style='list-style-type:disc'>The need for novel predictive biomarkers and targeted drug therapeutics is essential.</li></ul><li style='list-style-type:disc'>Current Focus:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Employing Thermal Proteome Profiling and in-depth proteogenomics.</li><li style='list-style-type:disc'>Aiming to identify novel predictive, resistant, and therapeutic biomarkers for more effective personalized therapy in pediatric ALL.</li></ul><li style='list-style-type:disc'>Significance of Biomarkers:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Seeking biomarkers to improve predictions of treatment response and resistance.</li><li style='list-style-type:disc'>Striving to minimize the side effects of broad cytotoxic therapy and enhance targeted drug therapeutics.</li></ul><li style='list-style-type:disc'>Long-Term Impact:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>The ultimate goal is to improve the overall efficacy and quality of life for pediatric ALL patients through innovative proteomics approaches and personalized therapeutic strategies.</li></ul></ul>",
        "researchLinks": ["https://doi.org/10.1038/s43018-021-00259-9","https://doi.org/10.1038/s41467-022-29224-5","https://doi.org/10.1038/s41467-022-33385-8"]
    },
    {
        "researchTitle": "Molecular Tumor Board Portal",
        "researchLead": "David Tamborero",
        "researchImage": "logo-portrait-karolinska-mtb-dark.svg",
        "researchDescription": "<ul style='padding-left:20px'><li style='list-style-type:disc'>Objective:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Support clinical decision-making through the interpretation of omics data from patient tumors.</li><li style='list-style-type:disc'>Contribute to precision oncology by accurately discovering and interpreting tumor alterations for diagnosis, prognosis, and therapy selection.</li></ul><li style='list-style-type:disc'>Challenges:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Current knowledge on tumor alterations is scattered across disparate resources with varying data models.</li><li style='list-style-type:disc'>Many observed tumor variants have unknown significance, requiring additional computational tools for relevance estimation.</li></ul><li style='list-style-type:disc'>Solution:</li><ul style='padding-left:20px'><li>Developed the Molecular Tumor Board (MTB) portal.</li><li style='list-style-type:disc'>Acts as a platform for annotating and distributing tumor omics data.</li><li style='list-style-type:disc'>Utilizes state-of-the-art knowledge bases and bioinformatics tools to interpret biological and clinical significance based on distinct levels of evidence.</li></ul><li style='list-style-type:disc'>Benefits of MTB Portal:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Ensures clinical decisions (e.g., inclusion in clinical trials, off-label use of anti-cancer drugs, genomic counseling) rely on the most updated and comprehensive knowledge available.</li><li style='list-style-type:disc'>Establishes a centralized framework for annotating and collecting results, contributing to the generation of novel knowledge for future patient decisions.</li></ul><li style='list-style-type:disc'>Current Usage:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Utilized by the seven leading European oncology centers forming the Cancer Core Europe.</li><li style='list-style-type:disc'>Currently supports genomic data analyses.</li></ul><li style='list-style-type:disc'>Future Goals:</li><ul style='padding-left:20px'><li style='list-style-type:disc'>Aim to implement interpretation of other omics data types, including transcriptomics, proteomics, and ex-vivo drug testing results.</li><li style='list-style-type:disc'>Continuously enhance the capabilities of the MTB portal to provide comprehensive support for clinical decision-making in precision oncology.</li></ul></ul>",
        "researchLinks": ["https://doi.org/10.1038/s41591-020-0969-2","https://doi.org/10.1038/s43018-022-00332-x"]
    }
]